src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -42,8 +42,8 @@ public static IEnumerable<Example> Examples
             {
                 yield return new Example(Resources.Example_UpdateCommand_SearchAndUpdateVersionAndInstallerURL, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "<InstallerUrl1>", "<InstallerUrl2>" }, Version = "<Version>" });
                 yield return new Example(Resources.Example_UpdateCommand_SaveAndPublish, new UpdateCommand { Id = "<PackageIdentifier>", Version = "<Version>", OutputDir = "<OutputDirectory>", GitHubToken = "<GitHubPersonalAccessToken>" });
-                yield return new Example(Resources.Example_UpdateCommand_OverrideArchitecture, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "<InstallerUrl1>|<InstallerArchitecture>" }, Version = "<Version>" });
-                yield return new Example(Resources.Example_UpdateCommand_OverrideScope, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "<InstallerUrl1>|<InstallerScope>" }, Version = "<Version>" });
+                yield return new Example(Resources.Example_UpdateCommand_OverrideArchitecture, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "“<InstallerUrl1>|<InstallerArchitecture>”" }, Version = "<Version>" });
+                yield return new Example(Resources.Example_UpdateCommand_OverrideScope, new UpdateCommand { Id = "<PackageIdentifier>", InstallerUrls = new string[] { "“<InstallerUrl1>|<InstallerScope>”" }, Version = "<Version>" });

```

Comment:
To display the quotes in the `wingetcreate update --help` helptext, I had to use a unicode characters `“ ”`. The Example class does not display double quotes / single quotes even after escaping them, so I had to use these unicode characters. They are still interpreted correctly in a shell like PowerShell

https://github.com/microsoft/winget-create/assets/88161975/fbb4ea96-68ed-47ca-aa70-f1aed5c4b284